data/org.freedesktop.portal.GlobalShortcuts.xml

```diff
@@ -183,6 +183,21 @@
       <arg type="o" name="request_handle" direction="out"/>
     </method>
 
+    <!--
+        ConfigureShortcuts:
+        @session_handle: Object path for the :ref:`org.freedesktop.portal.Session` object representing the session
+        @options: Vardict with optional further information
+
+        Request showing a configuration UI so the user is able to conigure all shortcuts of this session.
```

Comment:
Needs a `parent_window` arg, if this is intended to show UI.

Reply:
I was imagening it would open the Gnome Settings page or Plasma System Settings page in a non modal/blocking dialog way so was not sure if it needed a parent_window.   Added it now

Reply:
When opening systemsettings we also want an xdg activation token in case systemsettings is already open and needs to be raised

Reply:
Added activation_token support.